app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Kitty Items",
+  "description": "A complete NFT marketplace built with Cadence, Flow's resource-oriented smart contract programming language.",
+  "repository": "https://github.com/onflow/kitty-items",
+  "logo": "https://cdn.shopify.com/s/files/1/1061/1924/products/Happy_cat_emoji_icon_87ab6b39-5f08-48a5-a89a-1f5d3312e159_large.png?v=1571606090",
+  "keywords": [
+    "Cadence",
+    "Flow",
+    "NFT",
+    "Marketplace"
+  ],
+  "env": {
+    "SECRET_TOKEN": {
+      "description": "Example",
+      "generator": "example"
+    }
+  },
```

Comment:
Is this just a placeholder?

Should the `env` contain the `FLOW_ADDRESS` and `FLOW_PRIVATE_KEY`?

Reply:
Yes, this is incomplete. We'd need to fill in these with the actual env vars for the 'one-button-deploy'

Reply:
You can specify vars, but leave them blank so user fils them in at deploy time.